modules/ossm-vs-istio.adoc

```diff
@@ -10,27 +10,31 @@ An installation of {ProductName} differs from upstream Istio community installat
 The current release of {ProductName} differs from the current upstream Istio community release in the following ways:
 
 [id="ossm-multi-tenant-install_{context}"]
-== Multi-tenant installations
+== {ProductName} control plane
```

Comment:
Control Plane should probably be capitalized because the product name is also capitalized.

Reply:
The product name is capitalized, but we use sentence case titles at Red Hat.